README.md

```diff
@@ -1,11 +1,20 @@
 # ConfigMap & Secret replication for Kubernetes
 
-[![Docker Repository on Quay](https://quay.io/repository/mittwald/kubernetes-replicator/status "Docker Repository on Quay")](https://quay.io/repository/mittwald/kubernetes-replicator)
 [![Build Status](https://travis-ci.org/mittwald/kubernetes-replicator.svg?branch=master)](https://travis-ci.org/mittwald/kubernetes-replicator)
 
 This repository contains a custom Kubernetes controller that can be used to make
 secrets and config maps available in multiple namespaces.
 
+## Contents
+
+1. [Deployment](#deployment)
+    1. [Using Helm](#using-helm)
+    1. [Manual](#manual)
+1. [Usage](#usage)
```

Comment:
`2.`

Reply:
Not necessary. Markdown renders ordered lists automatically.